compiler/parser.mly

```diff
@@ -116,6 +117,24 @@ list:
   | expr                                  { [$1] }
   | list COMMA expr                       { $3 :: $1 }
 
+
+/* Distributions */
+dist_func:
+  | LCAR expr COMMA expr DDELIM expr VBAR
+    { {
+      min = $2;
+      max = $4;
+      dist_func = $6;
+    } }
+
+dist:
+  | LCAR expr COMMA expr RCAR
+    { {
+      min = $2;
+      max = $4;
+      dist_func = Ast.Id("uniform");
```

Comment:
is this going to overwrite the `uniform` function we use in the standard library?